static/js/attachments_ui.js

```diff
@@ -49,16 +52,20 @@ function set_upload_space_stats() {
     $("#attachment-stats-holder").html(rendered_upload_stats_html);
 }
 
-function delete_attachments(attachment) {
-    const $status = $("#delete-upload-status");
-    channel.del({
-        url: "/json/attachments/" + attachment,
-        error(xhr) {
-            ui_report.error($t_html({defaultMessage: "Failed"}), xhr, $status);
-        },
-        success() {
-            ui_report.success($t_html({defaultMessage: "Attachment deleted"}), $status);
-        },
+function delete_attachments(attachment, file_name) {
+    const html_body = render_confirm_delete_attachment();
+
+    if (file_name.length > 30) {
+        file_name = file_name.slice(0, 30) + "...";
+    }
```

Comment:
This sort of JS implementation is unreliable; because different characters have a different width, it'll look silly for one of `llllllllllllllllllllllllllllllllll` and `mmmmmmmmmmmmmmmmmmmmmmmmmm`.

Reply:
I went with this approach since I wasn't sure of a better approach that would work well in other languages too.